inc/networkinventory.class.php

```diff
@@ -367,11 +367,33 @@ public function run($jobstate)
         $current = $jobstate;
         $agent->getFromDB($current->fields['agents_id']);
 
-        $ip = current(PluginGlpiinventoryToolbox::getIPforDevice(
+        $ips = current(PluginGlpiinventoryToolbox::getIPforDevice(
             $jobstate->fields['itemtype'],
             $jobstate->fields['items_id']
         ));
```

Comment:
The loop `foreach ($ips as &$ip) {` will not work if you get the current element here.
```suggestion
        $device_ips = PluginGlpiinventoryToolbox::getIPforDevice(
            $jobstate->fields['itemtype'],
            $jobstate->fields['items_id']
        );
        $ip = current($ips);
```